python/cxx/deck_keyword.cpp

```diff
@@ -56,11 +59,80 @@ struct DeckRecordIterator
     }
 };
 
+
+bool is_number(const std::string& s)
+{
+    return !s.empty() && std::find_if(s.begin(), 
```

Comment:
I guess this could be done with `std::stod` and `try - excepct`

Reply:
stod produced double, which is not needed here. 
stoi produces integer, but does not cast exception for illegal values like "44re" and such.

Reply:
But stod is ok also for an integer. So for validation I would say stod + try except should be OK.